content/docs/using-github-directly/index.mdx

```diff
@@ -0,0 +1,303 @@
+
+![Copying the resource template folder](images/DC_CapEvent_img3_ingreen.png)
+
+Within this file, you will write all the content you want to include in your captured event. If you need a little more assistance in how to format this, or to include certain elements, such as videos, images, or tables, please take a look at our ‘Writing in Markdown’ guide in the ‘GitHub Workflow section.
+
+When you have your content in the index.mdx file, save it (although we also recommend saving this iteratively in your GitHub editor as a matter of course), and if you are running a local version of DARIAH-Campus in your browser you should see your event appear in the ‘Events’ page, as in here:
+
+![Copying the event template folder](images/DC_CapEvent_img4_inBrowser.png)
+
+Once you are happy with your 'Captured Event', you can ‘commit’ it via GitHub to the DARIAH-Campus team.
+
+## Creating content
+
+### Setting the Metadata for Citation
+
+The metadata of your resource is important as it forms the basis of the **citation** recommendation that you will see at the top of all the training resources. In order to ensure that this citation is correct, you will need to make sure you have the following information to hand:
+
+- The Title of Your Resource
+- The names of any authors, contributing authors or editors of your resource
+- The 'type' of training resource it is, e.g. 'training module', 'video'
+- The URL for where the resource is currently located. This is called the '**remoteUrl**' in the metadata (this only applies for External Resources)
+- The date your training resource was originally published. This is called the '**remotePublicationDate**' in the metadata (this only applies for External Resources)
+- The name of the organisation, or project that your resource comes from e.g. 'PARTHENOS-EU'. This is called the '**remotePublisher**' in the metadata (this only applies for External Resources)
+
+You can see how these metadata items fit into the citation below: ![DARIAH-Campus Citation Fields](images/dariah-campus-citation-fields.jpeg)
+
+The specific metadata that is required for your resource type is given at the top of all the index.mdx files in our template folders (see above), and it also comes with some guidance notes to help you find the information you need. There are more metadata fields for you to fill in, but these metadata fields above are particularly important for the citation.
+
+### Writing in Markdown
+
+DARIAH-Campus content needs to be written in Markdown. This can seem intimidating if you have not done it before, but there are many online guides to getting started with Markdown. For instance, Programming Historian have an excellent introduction, [Getting Started with Markdown](https://programminghistorian.org/lessons/getting-started-with-markdown).
+
+Text must be provided in a text editor such as Atom, TextWrangler, TextEdit, or Notepad++ rather than in a word processor such as MS Word.
```

Comment:
```suggestion
Text must be provided in a text editor such as Atom, VS Code, TextWrangler, TextEdit, or Notepad++ rather than in a word processor such as MS Word.
```